### PR TITLE
Podcast ep11

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Added page for podcast episode 11
- Added image for podcast 11

### CODE REVIEW

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- Recently [Sarus](https://sarus.readthedocs.io/en/stable/) was [added to Nextflow](https://github.com/nextflow-io/nextflow/pull/3470) as a container engine.
+- Sarus is an OCI-compatible container engine for HPC.
+- Sarus is an alternative container runtime to Docker.
+    - Sarus works by converting Docker images to a common format that can then be distributed and launched on HPC systems.
+- Similar deployment options to other container engines using Nextflow. 
+
+#### New stageOutMode directives
+
+- Two new `stageOutMode` directive (`rclone` and `fcp`) have been added to Nextflow.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:29]:
```suggestion
- Two new `stageOutMode` options (`rclone` and `fcp`) have been added to Nextflow.
```

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+#### The Sarus container engine
+
+- Recently [Sarus](https://sarus.readthedocs.io/en/stable/) was [added to Nextflow](https://github.com/nextflow-io/nextflow/pull/3470) as a container engine.
+- Sarus is an OCI-compatible container engine for HPC.
+- Sarus is an alternative container runtime to Docker.
+    - Sarus works by converting Docker images to a common format that can then be distributed and launched on HPC systems.
+- Similar deployment options to other container engines using Nextflow. 
+
+#### New stageOutMode directives
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:27]:
```suggestion
#### New stageOutMode options
```

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - Handle the most common use cases of cp with much higher performance.
+    - `fcp` is optimized for systems with an SSD.
+- The new directives will improve options for file movement as they are staged from the scratch directory.
+
+#### GITHUB_TOKEN
+
+- Git has become the de-facto standard for source code version control systems.
+- Nextflow provides built-in support for Git and the most popular Git hosting platforms.
+- Nextflow does not require any special configuration to access public repositories but requires repository credentials to access private repositories.
+- The [recent Nextflow commit](https://github.com/nextflow-io/nextflow/commit/c4d39384c3536e6777bc0edcd9fd33f147aec462) adds the ability to use the `GITHUB_TOKEN` environmental variable.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:46]:
```suggestion
- The [recent Nextflow commit](https://github.com/nextflow-io/nextflow/commit/c4d39384c3536e6777bc0edcd9fd33f147aec462) adds the ability to use the `GITHUB_TOKEN` environment variable.
```

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - The environmental variable is used when no creds are found in the source code management file.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:47]:
```suggestion
    - The environment variable is used when no credentials are found in the source code management configuration file (`$HOME/.nextflow/scm`).
```

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - It is especially when accessing pipeline code from a GitHub action.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:48]:
```suggestion
    - It is especially useful when accessing pipeline code from a GitHub action.
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
Pretty good! Just some minor things

**Comment by christopher-hakkaart** [comment]
Thanks for the comments @mribeirodantas

**Review by mribeirodantas — APPROVED** [review]

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    - The `stageOutMode` directive defines how output files are staged-out from the scratch directory to the process work directory.
+- [rclone](https://rclone.org/)
+    - Has been described as "The Swiss army knife of cloud storage".
+    - A command-line program to manage files on cloud storage.
+    - Over 40 cloud storage products support `rclone`.
+- [fcp](https://github.com/Svetlitski/fcp)
+    - A significantly faster alternative to the classic Unix `cp` command.
+    - Handle the most common use cases of cp with much higher performance.
+    - `fcp` is optimized for systems with an SSD.
+- The new directives will improve options for file movement as they are staged from the scratch directory.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:39]:
```suggestion
- The new options will improve file movement as there are new ways for the files to be staged from the scratch directory in different circumstances.
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

File: content/podcast/2023/ep11_sarus_out_git.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    - The `stageOutMode` directive defines how output files are staged-out from the scratch directory to the process work directory.
+- [rclone](https://rclone.org/)
+    - Has been described as "The Swiss army knife of cloud storage".
+    - A command-line program to manage files on cloud storage.
+    - Over 40 cloud storage products support `rclone`.
+- [fcp](https://github.com/Svetlitski/fcp)
+    - A significantly faster alternative to the classic Unix `cp` command.
+    - Handle the most common use cases of cp with much higher performance.
+    - `fcp` is optimized for systems with an SSD.
+- The new options will improve file movement as there are new ways for the files to be staged from the scratch directory in different circumstances.
```
Comment (mribeirodantas) [inline content/podcast/2023/ep11_sarus_out_git.md:39]:
```suggestion
- The new options will improve file management as there are new ways for the files to be staged from the scratch directory in different circumstances.
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

**Review by mribeirodantas — APPROVED** [review]

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**